### PR TITLE
Add gif to usage docs

### PR DESCRIPTION
**Pull request by abravalheri**
Related to the discussion in #477 .

I think once the PR is merged into master and deployed in RTD, I can also try to add it to the README.

[This is how it is currently looking like](https://pyscaffold--478.org.readthedocs.build/en/478/usage.html)

I have also added a link for a slightly longer version on asciinema.

### CODE REVIEW

**Comment by FlorianWilhelm** [comment]
Would it make sense to have between the `tox` block and the actual gif, some more sentences saying that this is a demonstration of `putup` using the experimental interactive mode and setting up a simple project named `myproj` with `--cirrus` then running `tox` and building it with `tox -e build`, etc.?

**Comment by abravalheri** [comment]
Thank you very much for the suggestion @FlorianWilhelm, I tried to add something in the latest commits.

Do you think it makes the text a bit repetitive (it feels a bit out of place).
 
There is also the possibility of placing the gif inside the `tip` box that advertises the interactive feature **or** in the examples section under the `putup -i my_little_project` example...

File: docs/usage.rst [hunk 1]
```diff
@@ -51,6 +51,15 @@ have tox_ installed and see what we have prepared for you out of the box::
    tox -e publish -- --repository pypi  # to release your package to PyPI
    tox -av  # to list all the tasks available
 
+The following figure demonstrates the usage of `putup` with the new experimental
+interactive mode for setting up a simple project named `myproj`.
+It uses the `--cirrus` flag to add CI support (via `Cirrus CI`_), and
+tox_ to run automated project tasks.
```
Comment (FlorianWilhelm) [inline docs/usage.rst:57]:
```suggestion
tox_ to run automated project tasks like building the `myproj` package.
```

**Comment by FlorianWilhelm** [comment]
@abravalheri, I think it looks really good. It's a concrete example of what was introduced before. lgtm.

**Pull request merged**